actors/runtime/runtime_util.go

```diff
@@ -64,7 +65,7 @@ func RT_MinerEntry_ValidateCaller_DetermineFundsLocation(rt Runtime, entryAddr a
 		return ownerAddr
 	} else {
 		if entrySpec == MinerEntrySpec_MinerOnly {
-			rt.AbortArgMsg("Only miner entries valid in current context")
+			rt.Abort(exitcode.ErrPlaceholder, "Only miner entries valid in current context")
```

Comment:
use ErrForbidden instead?

Reply:
This one indicates a programming error, not a caller error. I intend to rewrite it to be unnecessary, i.e make the error impossible.